packages/components/src/components/date-picker/DatePicker.tsx

```diff
@@ -500,7 +501,9 @@ class DatePicker extends Component<
 
     return (
       <div
-        role="tooltip"
+        role="dialog"
+        aria-modal="true"
+        aria-label={id === 'ADVANCED_SEARCH_SELECT_MESSAGE_START_DATE' ? 'Start DatePicker' : 'End DatePicker'}
```

Comment:
This is specific for start and end date picker. Will it read wrong when the date picker does not use for start or end date purpose?